test/baseclass.py

```diff
@@ -3,6 +3,7 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 #
 
+import dson
```

Comment:
So here's where it's probably annoying.  With multiple formats for config files, I want the test suite to test all possibilities.  I'd probably just make new environment variables that the Makefile can use.  Like RPMINSPECT_YAML, RPMINSPECT_DSON, and RPMINSPECT_JSON.  Then baseclass and other can figure out what to do from that.